packages/instantsearch.js/src/components/RefinementList/__tests__/RefinementList-test.tsx

```diff
@@ -692,5 +693,57 @@ describe('RefinementList', () => {
 
       expect(toggleRefinement).toHaveBeenCalledTimes(0);
     });
+
+    it('should keep focus on toggled input between re-renders', () => {
```

Comment:
this test should be in cts as well so it won't be reintroduced when we migrate components.